detox/src/configuration/composeSessionConfig.js

```diff
@@ -41,12 +48,37 @@ async function composeSessionConfig(options) {
     session.debugSynchronization = +cliConfig.debugSynchronization;
   }
 
+  if (isCloudSession) {
```

Comment:
@pb2323 Check if build, project and session name validation should check for empty string.